ds_judgements_public_ui/sass/includes/_govuk_overrides.scss

```diff
@@ -101,3 +101,20 @@
   background-color: #fef8df;
   padding: calc($spacer-unit * 2) calc($spacer-unit * 2);
 }
+
+.govuk-date-input__label {
+  font-size: 0.8rem;
+  margin-top: 0.25rem;
+}
+
+.govuk-date-input__input {
+  border: 2px solid #555;
+}
+
+.govuk-date-input__item {
+  margin-right: 0.5rem;
+}
+
+.govuk-form-group {
+  margin-bottom: 0;
+}
```

Comment:
We'll need to check these don't clash with overrides we introduced in the licence form once they're both merged, but no changes needed now